src/App.tsx

```diff
@@ -1,9 +1,126 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import 'date-fns';
+import './index.css';
+import Grid from '@material-ui/core/Grid';
+import FileUpload from './components/fileUpload/FileUpload';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import {CheckboxesGroup} from './components/inputs/checkboxes';
+import {RadioButtonsGroup} from './components/inputs/radioButtons'
+import Button from './components/inputs/calculateButton';
+import {MeasurementStart, MeasurementEnd} from './components/inputs/datePicker'
+
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex-grow',
+      flexDirection: 'column',
+      alignItems: 'center',
+      
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+    },
+    formControl: {
+      margin: theme.spacing(1),
+    },
+    textField: {
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
+      width: 200,
+    },
+    container: {
+      display: 'flex-grow',
+     
+    },
+    
+  }),
+);
+
+
+
 
 function App() {
+  const classes = useStyles();
+  const [file, setFile] = useState<string | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [results, setResults] = useState<null>(null);
```

Comment:
Is this state variable the result of execution? If so, I would make its type `any` for now
```suggestion
  const [results, setResults] = useState<any>(null);
```